web/pdf_page_view.js

```diff
@@ -40,6 +40,9 @@ var RenderingStates = pdfRenderingQueue.RenderingStates;
 
 var TEXT_LAYER_RENDER_DELAY = 200; // ms
 
+var CANVAS_RENDERER = 'canvas';
+var SVG_RENDERER = 'svg';
```

Comment:
I quick drive-by comment, but would it perhaps make sense to use an enumeration here instead, similar to many other places in the code? E.g.
```javascript
var Renderer = {
  CANVAS: 0, // Default
  SVG: 1,
};
```
Or it you need/want the content to be `string`s (e.g. for passing that in from `app.js`), it may still be clearer with e.g.
```javascript
var Renderer = {
  CANVAS: 'canvas', // Default
  SVG: 'svg',
};
```